installer/mac/ChainCore/Config.swift

```diff
@@ -7,30 +7,27 @@ class Config: NSObject {
 
     /// Checks if the port is in use by another process.
     static func portInUse(_ port: UInt) -> Bool {
-        let sock = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP)
-        if sock <= 0 {
-            return false
-        }
+        var output : [String] = []
+
+        let task = Process()
+        task.launchPath = "/usr/sbin/lsof"
```

Comment:
Would it be better to have cored exit with a particular status code if the port is in use? And then have the app just try to start it and handle the error when it doesn't work?

Just throwing it out as an option.